### PR TITLE
EMBCESSMOD-5139 EMBCESSMOD-5141 EMBCESSMOD-5142 EMBCESSMOD-5233: Self serve Interac E-Transfer, review, submission modal, validations 

### PR DESCRIPTION
**Pull request by HemanthKona**
## Self serve Interac E-Transfer, review, submission modal, validations
- Setup submission success dialog
- setup contact email form controls
- setup review form controls
- setup compare fields validator
- Fix showing error messages and loading spinner

### CODE REVIEW

File: registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
     Error: Unable to retrieve Draft Support information.
   </div>
 }
+@if (submitSupportError) {
+  <div style="display: block; color: white; background-color: orange; padding: var(--size-2); font-size: var(--size-3)">
+    Error: Unable to submit Support information.
+  </div>
+}
+@if (calculateTotalsError) {
+  <div style="display: block; color: white; background-color: orange; padding: var(--size-2); font-size: var(--size-3)">
```
Comment (ytqsl) [inline registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html:12]: shouldn't this be moved to a css class? it's being repeated multiple times

File: registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html
Comment (ytqsl) [inline registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html:1]: can we split these to sub components?

File: registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html (hunk 1 above)
Comment (HemanthKona) [reply registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html:12]: Yes, will fix that.

File: registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html
Comment (HemanthKona) [reply registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html:1]: Yes, I'll refactor it.

File: registrants/src/UI/embc-registrant/src/app/core/components/dialog-components/self-serve-submissoin-dialog/self-serve-submission-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+</div>
+
+<mat-dialog-content>
+  <h2 style="font-size: var(--size-3); font-weight: bold">Next Steps</h2>
+  <p>1. <b>Amount:</b> $ {{ data?.totalAmount }}</p>
+  <p>
+    @switch (data?.notificationPreference) {
+      @case (ETransferNotificationPreference.Email) {
+        <div style="display: flex">
+          <div style="width: 240px">2. <b>e-Transfer Notification:</b></div>
```
Comment (ytqsl) [inline registrants/src/UI/embc-registrant/src/app/core/components/dialog-components/self-serve-submissoin-dialog/self-serve-submission-dialog.component.html:20]: move styles to classes and make responsive

File: registrants/src/UI/embc-registrant/src/app/feature-components/self-serve-support/self-serve-support-form.component.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+          }
+        </div>
+      </mat-step>
+      <mat-step [stepControl]="eTransferDetailsForm">
+        <ng-template matStepLabel>e-Transfer Details</ng-template>
+        @if (eTransferDetailsForm) {
+          <app-self-serve-support-interac-e-transfer-form [eTransferDetailsForm]="eTransferDetailsForm">
+          </app-self-serve-support-interac-e-transfer-form>
+        }
+        <div style="display: flex; gap: 16px; justify-content: space-between">
```
Comment (ytqsl) [inline registrants/src/UI/embc-registrant/src/app/feature-components/self-serve-support/self-serve-support-form.component.html:72]: move styles to classes

**Review by ytqsl — CHANGES_REQUESTED** [review]

**Review by ytqsl — APPROVED** [review]

**Pull request merged**